joomla/joomla-cms
# [4.2] Enable use of Form::process() in FormModel

### Summary of Changes

A couple years ago @Hackwar added a cool feature #12414, which allows to run Field::postProcessing().
However it was not enabled in our FormModel class, and so never used.

This PR enable this feature for FormModel.


### Testing Instructions

Apply patch, edit any content, and make sure all works.

Then edit https://github.com/joomla/joomla-cms/blob/05d3b143887b4ef5286f80c388118b8e7b5b6a59/libraries/src/Form/FormField.php#L1305-L1308

Add `dd('Works!');` before `return`. And try to save any content again.


### Actual result BEFORE applying this Pull Request

Content saved, that means `postProcess()` was ignored.


### Expected result AFTER applying this Pull Request

You got white page with `Works!` message, that means `postProcess()` was called.


### Documentation Changes Required

As part of #12414
